lib/path/default.nix

```diff
@@ -271,6 +271,46 @@ in /* No rec! Add dependencies on this file at the top. */ {
               second argument: "${toString path2}" with root "${toString path2Deconstructed.root}"'';
         joinRelPath components;
 
+  /*
+  Split the filesystem root from a path.
+  The result is an attribute set with these attributes:
+
+  - `root`: The filesystem root of the path, meaning that this directory has no parent directory.
+  - `subpath`: The normalised subpath string that leads to the given path from the filesystem root, see `lib.path.subpath.normalise`
```

Comment:
_subpath string_ should be defined somewhere and linked.

I don't think we have this yet, so ignore the rest of this comment if we do.

It would be nice to have introductory docs between the title and the functions generated by nixdoc.

Maybe nixdoc could generate no title at all, and then we can just stick our own document in front with `cat`.

Don't care much about how it's implemented, but we need a definition.

Reply:
Ohh, now that @pennae made the `lib` rendering pipeline actually interpret the markdown correctly, we can actually link to these now! I'll make this change, should also be doing it retroactively for the other `lib.path` docs (in a separate PR).

Reply:
The code now links to `normalise` and `append` using proper markdown. I think this should be resolved with that.

Reply:
> me: True, it's not _necessary_ for everyone to

Looks like this comment got stuck in an unsubmitted review. There wasn't a point to be made anymore...
Also very odd that github will render it as part of a conversation as if it was visible all this time??